src/main/scala/com/actian/spark_vector/datastream/VectorEndpointConf.scala

```diff
@@ -13,15 +13,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.actian.spark_vector.writer
+package com.actian.spark_vector.datastream
 
 import com.actian.spark_vector.vector.VectorJDBC
 
-/** Configuration for writing, one entry for each Vector end point expecting data */
-case class WriteConf(vectorEndPoints: IndexedSeq[VectorEndPoint]) extends Serializable
+/** Configuration for reading/writing end points - one entry for each Vector end point expecting data */
```

Comment:
you can't write end points, can you? It's either read/write end points or just reading/writing.